### ISSUE TITLE
Include "Success Criteria" (or "Exit Criteria")

### ISSUE THREAD

**Issue by csarven** [solid/specification#373]
Example "Success Criteria": https://github.com/solid/process/blob/main/notifications-panel-charter.md#success-criteria

Or an "Exit Criteria" when typically transitioning from a ~CR to ~PR as per W3C Process, e.g, https://www.w3.org/TR/ldn/#exit-criteria

**Comment by kjetilk** [comment]
Yeah, I think that we either need something that resembles that (possibly something simpler), or we need to transition to a W3C IG or WG structure, in which case that will be part of the process. I think it makes sense to reach 1.0 first though.

**Comment by michielbdejong** [comment]
Good stuff! Apart from criteria about how often a given spec was implemented, for the Solid Project we may want to add requirements about how many competing specs aim to achieve the same goal. For instance, ACP aims to obsolete WAC and SAI aims to obsolete type indexes. They should not go from 'experiment' to 'accepted' unless the spec they replace also goes to 'superseded by ...', otherwise our protocol stack becomes an unstructured protocol pile.
